app-admin/keepassxc/keepassxc-2.5.4.recipe

```diff
@@ -0,0 +1,145 @@
+KeePassXC is a community fork of KeePassX which aims to incorporate stalled \
+pull requests, features, and bug fixes that have never made it into the main \
+KeePassX repository."
+HOMEPAGE="https://keepassxc.org/"
+COPYRIGHT="
+	2010-2012, Felix Geyer
+	2011-2012, Florian Geyer
+	2012, Tobias Tangemann
+	2007, Trolltech ASA
+	2012, Intel Corporation
+	2012, Nokia Corporation and/or its subsidiary(-ies)
+	2000-2008, Tom Sato
+	2013, Laszlo Papp
+	2013, David Faure
+	2016-2020, KeePassXC Team
+	"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/keepassxreboot/keepassxc/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="ebb27af867548f765d2b3aeb3937ad34f429159f7ce12af7a5db6a744997c5aa"
+#PATCHES="keepassxc-$portVersion.patchset"
+ADDITIONAL_FILES="keepassxc.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	keepassxc$secondaryArchSuffix = $portVersion
+	cmd:keepassxc_cli
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libargon2$secondaryArchSuffix
+	lib:libgcrypt$secondaryArchSuffix
+	lib:libgpg_error$secondaryArchSuffix
+	lib:libmicrohttpd$secondaryArchSuffix
+	lib:libqrencode$secondaryArchSuffix
+	lib:libsodium$secondaryArchSuffix
+	lib:libQt5Concurrent$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5DBus$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Test$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libargon2$secondaryArchSuffix
+	devel:libgcrypt$secondaryArchSuffix
+	devel:libgpg_error$secondaryArchSuffix
+	devel:libmicrohttpd$secondaryArchSuffix
+	devel:libqrencode$secondaryArchSuffix
+	devel:libsodium$secondaryArchSuffix
+	devel:libQt5Concurrent$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5DBus$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Test$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:lrelease$secondaryArchSuffix >= 5
+	cmd:make
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+
+BUILD()
+{
+	mkdir -p build ; cd build
+
+	cmake .. -DCMAKE_INSTALL_PREFIX=$prefix \
+		-DCMAKE_INSTALL_DATAROOTDIR=$dataDir \
+		-DKEEPASSXC_BUILD_TYPE=Release \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DWITH_XC_NETWORKING=on \
+		-DWITH_XC_SSHAGENT=on \
+		-DWITH_XC_AUTOTYPE=on \
+		-DWITH_XC_UPDATECHECK=off \
+		-DWITH_TESTS=off \
+		-DCMAKE_CXX_FLAGS="-D_GNU_SOURCE"
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+#	TODO: adjust path for /boot/home/.config/keepassxc/keepassxc.ini
+#	TODO: replace icon with the one from keepassxc.svgz
+#	TODO: replace toolbar icons with https://github.com/HaikuArchives/ZumisIcons/tree/master/myhaiku/btoolbar
+	cd build
+	make install
+
+	appIconsDir=$dataDir/keepassxc/icons/application
```

Comment:
I suppose this line can be commented out as well. $appIconsDir appears to be never used.